File: Assignment Programs/Assignment33/program5.java
import java.util.*;

class Digit
{
	public int CountDiff(int iNo)
	{
		int iDigit = 0, iEven = 0, iOdd = 0;
		
		while(iNo != 0)
		{
			iDigit = iNo % 10;
			
			if((iDigit % 2) == 0)
			{
				iEven = iEven + iDigit;
			}
			else
			{
				iOdd = iOdd + iDigit;
			}
			
			iNo = iNo / 10;
		}
		return (iEven - iOdd);
	}
}

class program5
{
	public static void main(String args[])
	{
		int iValue = 0, iRet = 0;
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the number");
		iValue = sobj.nextInt();
	
		Digit dobj = new Digit();
		
		iRet = dobj.CountDiff(iValue);
	
		System.out.println("Multiplication of all digits is : "+iRet);
	}
}

File: Assignment Programs/Assignment6/program5.c
#include<stdio.h>
int CountDiff(int iNo)
{
	int iDigit = 0, iEvenSum = 0, iOddSum = 0, iDiff = 0;
	while(iNo != 0)
	{
		iDigit = iNo%10;
		if(iDigit%2 == 0)
		{
			iEvenSum = iEvenSum + iDigit;
		}
		else
		{
			iOddSum = iOddSum + iDigit;
		}
		iNo = iNo/10;
	}
	iDiff = iEvenSum - iOddSum; 
	return iDiff;
}
int main()
{
	int iValue = 0;
	int iRet = 0;
	printf("Enter number\n");
	scanf("%d",&iValue);
	iRet = CountDiff(iValue);
	printf("%d",iRet);
	return 0;
}